Source/It.cs

```diff
@@ -141,5 +141,35 @@ public static string IsRegex(string regex, RegexOptions options)
 			// But evaluated every time :)
 			return Match<string>.Create(value => re.IsMatch(value), () => It.IsRegex(regex, options));
 		}
+
+		/// <summary>
+		/// Allows for setting up open generic methods on Mock Fixtures
+		/// </summary>
+		public interface AnyType
+		{
+			/// <summary>
+			/// Will contain the specific object being compared inside an open generic method.
+			/// Used for checking inside It.Is&lt;It.AnyType&gt;(x => ((int)x.Object) == 2)
+			/// </summary>
+			object Object { get; set; }
```

Comment:
To be honest, everything that could be in the `It.Is<It.AnyType>` `Predicate`, could be done in the `Returns` method.

The only reason I added it was because it was in line with the other `It.Is<T>` predicates.

I guess it would only be useful if `Verify` for `It.AnyType` was implemented.

I have no issue dropping this overload (less things to maintain)